tests/docker/Dockerfile

```diff
@@ -142,8 +142,8 @@ RUN python3 -m pip install --upgrade --force pip && \
     python3 -m pip install --force --no-cache-dir -e /root/tests/
 
 # Install the OMB tool
-RUN git -C /opt clone https://github.com/redpanda-data/openmessaging-benchmark.git && \
-    cd /opt/openmessaging-benchmark && git reset --hard cde313756faa1ebd0114a35d6fc3b09198559738 && mvn package
+RUN git -C /opt clone https://github.com/bharathv/openmessaging-benchmark.git && \
```

Comment:
This is a temp change to showcase the e2e patch. Had to make some changes in the way we chart the metrics. You can find the patch [here](https://github.com/bharathv/openmessaging-benchmark/commit/aa943b284bf81796a968a87e788f9b2c0747de10), will update to official repos before merging.

Reply:
Don't forget to change it also in CDT setup
https://github.com/redpanda-data/vtools/blob/dev/qa/image/packer/install-deps.sh#L195